openspoor/visualisations/spoorkaart.py

```diff
@@ -0,0 +1,222 @@
+import pandas as pd
+import folium
+from shapely.geometry import point
+from typing import List, Optional, Dict, Tuple
+from pathlib import Path
+from urllib.parse import quote
+from ..utils.common import read_config
+
+config = read_config()
```

Comment:
global variables should be upper case for easy recognition as globals

Reply:
Good suggestion; this would need to change throughout the entire repo however. It would probably be better to address in a separate PR.